.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: jruby-9.2.17.0
-        bundler: 1
+        bundler: 2
```

Comment:
I think this can be omitted and it will pick latest

Reply:
https://github.com/ruby/setup-ruby/blob/b161cb92f0e2f28d992f04f38fff204d84132c47/action.yml#L16

Reply:
Verdad!